woocommerce/storefront
# Remove `.gitattributes` file and add excluded files to `composer.json`

Semi-Fixes #1760 - there will be steps to do after this PR is merged.

To summarise the issue, in #1674 we added a `.gitattributes` file. This excluded files from the _source archives_ while the intention was to exclude them only from the _built archives_. This PR removes that file, as it's not necessary. We're using composer to generate the built archive in the `postbuild` script. Files that should not be included in this archive have been added to the `composer.json` file.

We will probably need to go back and tag point releases to allow users to download source archives for the affected versions.

### How to test the changes in this Pull Request:


1. Run `npm run build` and inspect the generated archive. Ensure it doesn't contain development or config files such as `.gitignore`.
2. Upload the archive to a JN site and ensure it works properly.
3. General smoke test of Storefront functionality

### Changelog



> Fix – Resolved an issue where source archives downloaded from GitHub did not contain all files.
